examples/dao/setup_sigma_dao.sql

```diff
@@ -11,6 +11,16 @@ CREATE TABLE IF NOT EXISTS sigma_daos (
   asset_id BIGINT -- token id
 );
 
+-- create a sigma_dao_proposal table if does not exit already.
+CREATE TABLE IF NOT EXISTS sigma_dao_proposals (
+  id SERIAL PRIMARY KEY, -- auto increment id
+  addr BYTEA,
+  app BIGINT,
+  localstate JSONB,
```

Comment:
how about renaming to "details". Please add a link in a comment to a resource describing what are we going to store here (a link to a code or spec)

Reply:
The other table `account_app` also used the same column name. We are using trigger on that table update. I think keeping the same column name would be good idea.